src/view/components/selectorLearnMoreLink.styl

```diff
@@ -0,0 +1,3 @@
+.SelectorLearnMoreLink {
+  margin-right: 56px;
+}
```

Comment:
Hey, this isn't a big deal and I'm not suggesting you change this in your PR, but you might want to update your editor to insert a newline at the end of the files upon saving. In practice, it's not very important to our JavaScript use cases, but it was an important thing in C and has just continued as a "best practice". Thus, other people's editors will likely add a newline when they edit anything in this same file, which just causes unnecessary church later on in git.

Reply:
Done! and thanks for the tip.